src/shared/ui/Text/index.tsx

```diff
@@ -63,6 +64,7 @@ function Text({
       lineHeight={lineHeight}
       letterSpacing={letterSpacing}
       onClick={onClick}
+      {...(component ? { component } : {})}
```

Comment:
A tad more concise 😇 
```suggestion
      {...(component && { component })}
```